src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingServiceTest.java

```diff
@@ -195,6 +196,32 @@ void testFilingGenerationWhenSuccessfulWithoutTrustsAndWithIdentityChecks(Submis
         checkManagingOfficers(filing);
     }
 
+    @Test
+    void testFilingGenerationAllowedWhenEntityNameBlockToBeNull() throws IOException, URIValidationException, ServiceException, SubmissionNotFoundException, JSONException {
```

Comment:
Would rename this method slightly - `testFilingGenerationAllowedWhenEntityNameBlockIsNull`.

Reply:
Renamed this.